wiki/osu!academy/en.md

```diff
@@ -52,6 +52,6 @@ If you have any questions or comments, please forward them to [ztrot](https://os
 
 ## External Links
 
-- Source: <https://news.ppy.sh/post/70504430082/introducing-the-osu-academy>
+- Source: <https://osu.ppy.sh/home/news/2013-12-19-introducing-the-osu-academy>
```

Comment:
Shorten the url, and link the site with the word on the left(i.e. `Source`) because it violates the asc.
Do for every updated translations of this page.